composer.json

```diff
@@ -10,20 +10,23 @@
         }
     ],
     "require": {
+        "ezsystems/ezpublish-kernel": "~5.1 || ^6.7",
```

Comment:
7.0 should be added on master (2.0)

I’m assuming the depeneides are strict here, I know some bundles tried to have soft dependency on kernel to also be able to install on plain symfony